Improve wording on `Alternative` section (#23)
Kludex/no-optional: README.md

## Alternative

[`pyupgrade`](https://github.com/asottile/pyupgrade) is great but `no-optional` is better when you need runtime support like for `FastAPI` and `pydantic`.

> The reason being that `no-optional` just do the replacement. On the other hand, `pyupgrade` adds [`from __future__ import annotations`](https://peps.python.org/pep-0563/) for versions below Python 3.10.

Reviewer: ```suggestion
The reason being that `no-optional` just does the replacement. On the other hand, `pyupgrade` requires [`from __future__ import annotations`](https://peps.python.org/pep-0563/) for versions below Python 3.10.
```
